Sources/Async/Streams/EncodingStream.swift

```diff
@@ -0,0 +1,52 @@
+/// Capable of encoding futures.
+public protocol StreamEncoder: class {
+    /// Encodes a future to the encoder.
+    func encodeStream<O: OutputStream & ConnectionContext>(_ stream: O) throws where O.Output == Encodable
+}
+
+public final class EncodableStream: TransformingStream, ConnectionContext, Encodable {
```

Comment:
I don't think we need this class. See my comment on https://github.com/vapor/leaf/pull/84#pullrequestreview-85888525